src/components/probe/index.ts

```diff
@@ -144,14 +138,13 @@ function checkThresholdsAndSendAlert(
 }
 
 export function getProbeStatesWithValidAlert(
-  probeStates: ServerAlertState[],
-  isSymonMode: boolean
+  probeStates: ServerAlertState[]
 ): ServerAlertState[] {
   return probeStates.filter(
     ({ isFirstTime, shouldSendNotification, state }) => {
       // ignore first up event for non Symon mode
```

Comment:
remove this comment, i don't think it's accurate anymore, right?

Reply:
Comment removed.